elide-standalone/src/main/java/com/yahoo/elide/standalone/Util.java

```diff
@@ -53,28 +64,61 @@ public static EntityManagerFactory getEntityManagerFactory(String modelPackageNa
             options.put("javax.persistence.jdbc.password", "elide123");
         }
 
-        PersistenceUnitInfo persistenceUnitInfo = new PersistenceUnitInfoImpl("elide-stand-alone",
-                combineModelEntities(modelPackageName, includeAsyncModel), options);
+        dynamicEntityCompiler = new ElideDynamicEntityCompiler(dynamicConfigPath);
```

Comment:
This line should be inside the if block at L-68, otherwise even when the dynamic config is disabled it will try to generate the Config POJOs.